indico/client/request.py

```diff
@@ -51,3 +52,17 @@ class RequestChain:
 
     def requests(self):
         pass
+
+
+class Debouncer:
+    def __init__(self):
+        self.timeout = 0
+        self.max_timeout = 20
```

Comment:
I think 5 ok, like we we don't need to debounce that much

Reply:
updated to 5.